lib/drum.hoon

```diff
@@ -9,10 +9,23 @@
 ::::                                                    ::  ::
   ::                                                    ::  ::
 |%                                                      ::  ::
-++  drum-part  {$drum $1 drum-pith-1}                   ::
-++  drum-part-old  {$drum $0 drum-pith-0}               ::
+++  drum-part  {$drum $2 drum-pith-2}                   ::
+++  drum-part-old                                       ::
+  $:  $drum                                             ::
+      $%  {$1 drum-pith-1}                              ::
+          {$0 drum-pith-0}                              ::
```

Comment:
This line is no longer necessary if you have a legit `drum-part-old` case, and is crashing because `$%` bunts to the last variant.